Source/Core/Core/HW/DSPHLE/UCodes/AX.cpp

```diff
@@ -22,6 +22,17 @@ namespace DSP
 {
 namespace HLE
 {
+static bool HasLpf(u32 crc)
+{
+  switch (crc)
+  {
+  case 0x4e8a8b21:
```

Comment:
Could that file be referenced, or could the check be performed in that file instead (assuming that works with how the code is designed)?